iamaccessgroupsv2/iam_access_groups_v2_examples_test.go

```diff
@@ -23,11 +22,10 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
+	"time"
 
 	"github.com/IBM/go-sdk-core/v5/core"
 	"github.com/IBM/platform-services-go-sdk/iamaccessgroupsv2"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
```

Comment:
Why these have been removed? As you can see the builds failed due these missing dependencies.